package.json

```diff
@@ -11,20 +11,20 @@
     "dependencies": {
         "@adobe/asset-compute-events-client": "^1.0.1",
         "@adobe/jwt-auth": "^0.3.3",
-        "@adobe/node-fetch-retry": "^1.1.0",
-        "fs-extra": "^9.1.0",
-        "js-yaml": "^4.0.0",
+        "@adobe/node-fetch-retry": "^1.1.1",
+        "fs-extra": "^10.0.0",
```

Comment:
Is this going to result in a breaking change for us?

Reply:
No :)